code/processes/gateway.q

```diff
@@ -398,73 +402,79 @@ getserveridstype:{[att;typ]
   attype:`cross;
 
   servers:$[typ=`all;
-		exec serverid!attributes from .gw.servers where active;
-		exec serverid!attributes from .gw.servers where active, servertype=typ];
+    exec serverid!attributes from .gw.servers where active;
+    exec serverid!attributes from .gw.servers where active,servertype=typ];
 
   if[`besteffort in key att;
-	if[-1h=type att`besteffort; besteffort:att`besteffort];
-	att:delete besteffort from att;
+  if[-1h=type att`besteffort;besteffort:att`besteffort];
+  att:delete besteffort from att;
   ];
   if[`attributetype in key att;
-	if[-11h=type att`attributetype; attype:att`attributetype];
-	att:delete attributetype from att;
+  if[-11h=type att`attributetype;attype:att`attributetype];
+  att:delete attributetype from att;
   ];
 
   res:$[attype=`independent; 
-	getserversindependent[att;servers;besteffort];
-	getserverscross[att;servers;besteffort]];
+  getserversindependent[att;servers;besteffort];
+  getserverscross[att;servers;besteffort]];
```

Comment:
418 + 419 need additional indentation as they are only executed as a result of 417.